kubeflow/common/centraldashboard.libsonnet

```diff
@@ -83,6 +83,53 @@
     },  //service
     centralDashboardService:: centralDashboardService,
 
+    local centralDashboardIstioVirtualService = {
+      apiVersion: "networking.istio.io/v1alpha3",
+      kind: "VirtualService",
+      metadata: {
+        name: "centraldashboard",
+        namespace: params.namespace,
+      },
+      spec: {
+        hosts: [
+          "*",
+        ],
+        gateways: [
+          "kubeflow-gateway",
+        ],
+        http: [
+          {
+            match: [
+              {
+                uri: {
+                  prefix: "/_/",
```

Comment:
This is the weird frontend layout if it's `prefix /` instead of `prefix /_/`:
![image](https://user-images.githubusercontent.com/4784270/56516059-602bfb80-64ee-11e9-9553-03eee260c674.png)

Reply:
So in 0.5.0 using Ambassador, if I directly navigate to a route like `https://<host>/_/katib-dashboard`, there's no repetition of the landing page like the screenshot shows. We use the `/_/` convention to specify a client-side routing rule that loads the appropriate page in the iframe of the central dashboard. I'm not at all familiar with the way routing is configured in Istio but I assumed that any route that was not previously matched could be defaulted to be handled by the Central Dashboard. Does this mean that any other convention we use for client-side routing would also need to be reflected in the configuration here?

Reply:
`/_` is used by the dashboard and should not be handled by istio to route elsewhere, including any custom 404. (this behavior is controlled by the dashboard now)

### Explanation:
`/_` pages behavior: (ex. `/_/notebooks`)
- Load the dashboard
- Select the appropriate sidebar item
- Load the actual location of the target sub-app **in an iframe** (which in this example would be `/jupyter/`)

Reply:
somehow I can't reproduce this problem after I submitted this PR and deploy from HEAD...

please help review this PR, thanks: https://github.com/kubeflow/kubeflow/pull/3076